algorithms/CPlusPlus/Dynamic-Programming/longest-common-substring.cpp

```diff
@@ -0,0 +1,39 @@
+// s1: abcde
+// s2:abfce
+// output: 2(ab)
```

Comment:
add it at the end 
at the top add a user-friendly description of the program
as well as add time complexity of the program